pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -1015,13 +1028,6 @@
     "vendorSha256": null,
     "version": "2.0.2"
   },
-  "vthunder": {
```

Comment:
It still exists but it redirects to `thunder` and we'd need to override the build to get it to work as `vthunder`.